2020-01-Winter/1_linked_lists/main.cpp

```diff
@@ -0,0 +1,348 @@
+#include "Node.h"
```

Comment:
Can you move source files to a subdirectory? It helps isolate individual problems and keep the folder well-structured.